### PR TITLE
PyROOT fix

### PR DESCRIPTION
**Pull request by peremato**
BEGINRELEASENOTES
- Fine tune the latest fix to cope with new PyROOT to use old API for versions 6.20.X
ENDRELEASENOTES

### CODE REVIEW

**Comment by peremato** [comment]
Sorry, I made a small mistake with the versions. 6.20.0X > 6.20. Now I do 6.20.X < 6.21.

**Comment by rete** [comment]
Hi Pere,
thanks for the fix!
It seems you have conflicts in your branch.

File: src/python/pyLCIO/base/HandleExceptions.py [hunk 1]
```diff
@@ -12,6 +12,12 @@
 import inspect, sys, ROOT, pyLCIO
 import pyLCIO.exceptions.Exceptions
 
+# Cope with API change with ROOT version > 6.20
```
Comment (andresailer) [inline src/python/pyLCIO/base/HandleExceptions.py:15]:
```suggestion
# Cope with API change with ROOT version >= 6.21
```
To make the comment consistent with the code

**Comment by peremato** [comment]
> Hi Pere,
> thanks for the fix!
> It seems you have conflicts in your branch.

Probably the mistake is that I reuse the branch of the previous MR.

**Comment by petricm** [comment]
rebased in #77

**Pull request closed without merging**